helpers.py

```diff
@@ -0,0 +1,258 @@
+#============================
+#PACKAGES
+#============================
+
+import os
+import requests
+import traceback
+import time
+import numpy as np
+
+#============================
+#LOG HANDLER
+#============================
+def log(message):
+    try:
+        logHandler=open((os.getcwd()+"/log.log"), 'a')
+        print(message)
+        logHandler.write(message)
+        logHandler.write("\n")
+        logHandler.close()
+    except Exception:
+        print (traceback.format_exc())
+
+#============================
+#FILE READER
+#============================
+def readFile(fName):
+    try:
+        fileHandler=open((os.getcwd()+fName), 'r')
+        log("Loading "+fName+" ...")
+        rawFile=fileHandler.read()
+        fileHandler.close()
+        return [False,rawFile]
+    except Exception:
```

Comment:
This looks like a weird exception handling: you are masking the exception here. Why not just let it raise? Same comment applies to the other functions below.